### PR TITLE
adds docker build and push

### PR DESCRIPTION
**Pull request by tudorcodrea**

### CODE REVIEW

**Review by jozsef-l-sas — DISMISSED** [review]

**Review by alexandru-damian — APPROVED** [review]

File: .github/workflows/be_user_service.yml [hunk 1]
```diff
@@ -43,10 +43,22 @@ jobs:
          architecture: x64
 
       - name: Build with maven
-        run: mvn dependency:tree clean install
+        run: mvn clean install
```
Comment (alexandru-damian) [inline .github/workflows/be_user_service.yml]: The log should be verbose to find conflicts ..

**Review by alexandru-damian — DISMISSED** [review]

File: .github/workflows/be_user_service.yml (hunk 1 above)
Comment (tudorcodrea) [reply .github/workflows/be_user_service.yml]: added back the verbosecontent during build

**Review by tudorcodrea — COMMENTED** [review]
reintroduced verbose content during github actions build

**Review by tudorcodrea — COMMENTED** [review]
reintroduced verbose content during github actions build

**Review by jozsef-l-sas — APPROVED** [review]

**Review by jozsef-l-sas — APPROVED** [review]

**Pull request merged**